Error on chat message

```
18:16:12.4493 [ERROR]  Torch.Managers.NetworkManager: System.NullReferenceException: Object reference not set to an instance of an object.
   at Essentials.Commands.InfoModule.<>c__DisplayClass2_0.<MessageProcessing>b__0(InfoCommand i)
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source, Func`2 predicate)
   at Essentials.Commands.InfoModule.MessageProcessing(TorchChatMessage msg, Boolean& consumed)
   at Torch.API.Managers.MessageProcessingDel.Invoke(TorchChatMessage msg, Boolean& consumed)
   at Torch.Managers.ChatManager.ChatManagerServer.RaiseMessageRecieved(ChatMsg message, Boolean& consumed)
   at Torch.Managers.ChatManager.ChatManagerServer.ChatIntercept.Handle(UInt64 remoteUserId, CallSite site, BitStream stream, Object obj, MyPacket packet)
   at Torch.Managers.NetworkManager.OnEvent(MyPacket packet)
```

I assume this is caused by `EssentialsPlugin.Instance.Config.InfoCommands` being and trying to run FirstOrDefault on it?

https://github.com/TorchAPI/Essentials/blob/master/Essentials/Commands/InfoModule.cs#L28

--- a/Essentials/Commands/InfoModule.cs
+++ b/Essentials/Commands/InfoModule.cs
@@ -25,7 +25,11 @@ namespace Essentials.Commands
 
         private static void MessageProcessing(Torch.API.Managers.TorchChatMessage msg, ref bool consumed)
         {
-            var c = EssentialsPlugin.Instance.Config.InfoCommands.FirstOrDefault(i => i.Command.Equals(msg.Message));
+            var infoCommands = EssentialsPlugin.Instance.Config.InfoCommands;
+            if (infoCommands == null)
+                return;
+                
+            var c = infoCommands.FirstOrDefault(i => i.Command.Equals(msg.Message));
             if (c == null)
                 return;
             

